src/IRsend.h

```diff
@@ -157,6 +157,7 @@ enum gree_ac_remote_model_t {
   YBOFB,      // (2) Green, YBOFB2, YAPOF3
   YX1FSF,     // (3) Soleus Air window unit (Similar to YAW1F, but with an
               //     Operation mode of Energy Saver (Econo))
+  YX1F8F,     // Similiar to YX1FSF
```

Comment:
Anything that differs?
As a reader of just the comment I wonder why it is needed, hopefully this can be improved on to help other readers understand it. Maybe: "Mostly identical to YX1FSF but different identification"

Reply:
The hard part with explaining why the model is different is because the parts of the protocol that are identified and named in the code remain the same for this model. The only differences are that this particular remote has a different set of static "unknowns" than the other models, along with the addition of the ModelA bit needing to be set like the YAW1F remote.

I don't know if it adds benefit to explain that the model is "Similiar to YX1FSF, except for different unknowns". Seems it would add more confusion than clarity?

Reply:
How about "same as YAW1F, model bits differs"